src/core/core.tooltip.js

```diff
@@ -985,8 +1002,12 @@ var exports = Element.extend({
 		// Find Active Elements for tooltips
 		if (e.type === 'mouseout') {
 			me._active = [];
+			me._lastEvent = null;
 		} else {
 			me._active = me._chart.getElementsAtEventForMode(e, options.mode, options);
+			if (e.type !== 'click') {
+				me._lastEvent = e;
+			}
```

Comment:
To be safe(r), I'd use `me._lastEvent = e.type === 'click' ? null : e;` here too.